package.json

```diff
@@ -1,16 +1,20 @@
 {
   "name": "business-filings-ui",
-  "version": "7.3.27",
+  "version": "7.4.0",
   "private": true,
   "appName": "Filings UI",
   "sbcName": "SBC Common Components",
   "scripts": {
     "dev": "vite",
     "build": "vite build",
-    "preview": "vite preview",
-    "lint": "eslint . --ext .js,.ts,.vue",
+    "build-check": "vite build",
```

Comment:
I think this, and perhaps some of the others, might be needed by scripts. Anyway, they don't hurt to have here.